content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md

```diff
@@ -47,13 +47,13 @@ the container starts.
 
 1. Display information about the Pod:
 
-       kubectl get pod termination-demo
+        kubectl get pod termination-demo
```

Comment:
need to format lines like this as code explicitly.

Reply:
In general, using 
'''shell   
'''
But there have numlist, we should format like that. Besides, If we format there explicitly, doesn't work. Look https://deploy-preview-8837--kubernetes-io-master-staging.netlify.com/docs/tasks/debug-application-cluster/determine-reason-pod-failure/, it's correctly.

Reply:
Checked the preview. It is okay.